Formula/gtk-vnc.rb

```diff
@@ -1,8 +1,8 @@
 class GtkVnc < Formula
   desc "VNC viewer widget for GTK"
   homepage "https://wiki.gnome.org/Projects/gtk-vnc"
-  url "https://download.gnome.org/sources/gtk-vnc/1.2/gtk-vnc-1.2.0.tar.xz"
-  sha256 "7aaf80040d47134a963742fb6c94e970fcb6bf52dc975d7ae542b2ef5f34b94a"
+  url "https://download.gnome.org/sources/gtk-vnc/1.3/gtk-vnc-1.3.1.tar.xz"
```

Comment:
Don't know about version scheme but looks like all major Linux distros are using 1.3.x (https://repology.org/project/gtk-vnc/versions).

Should also fix Ventura build which is failing due to newer `meson` version.